app/src/main/java/com/babylon/wallet/android/presentation/wallet/WalletScreen.kt

```diff
@@ -224,6 +230,23 @@ private fun WalletAccountList(
                 style = RadixTheme.typography.body1HighImportance,
                 color = RadixTheme.colors.gray2
             )
+            Spacer(modifier = Modifier.height(RadixTheme.dimensions.paddingXLarge))
+            Text(
+                text = stringResource(R.string.homePage_totalValue).uppercase(),
+                style = RadixTheme.typography.body2Header,
+                color = RadixTheme.colors.gray2
+            )
+            Spacer(modifier = Modifier.height(RadixTheme.dimensions.paddingXXSmall))
+            TotalFiatBalanceView(
+                fiatPrice = state.totalFiatValueOfWallet,
+                isLoading = state.isWalletBalanceLoading,
+                currency = SupportedCurrency.USD,
```

Comment:
maybe we could have this stored via preference manager, and made USD by default. Unless we won't support other currencies for a long time

Reply:
that's maybe a good improvement, but not for now. 
This feature has grown dramatically and I think it's to leave it as it is.
Once we start supporting other currencies (no idea when, and was never mentioned in meetings) then we can refactor accordingly.